### ISSUE TITLE
Bootstrap 3 and CheckboxSelectMultiple: Checkboxes misaligned

### ISSUE THREAD

**Issue by nrogers64** [maraujop/django-crispy-forms#366]
Here is a test form:

```python
class TestForm(forms.Form):
    THREE_CHOICES = (
        ('1', 'Choice 1'),
        ('2', 'Choice 2'),
        ('3', 'Choice 3'),
    )

    text_field = forms.CharField()
    single_select = forms.ChoiceField(choices=THREE_CHOICES)
    multiple_select = forms.MultipleChoiceField(choices=THREE_CHOICES)
    single_checkbox = forms.BooleanField()
    multiple_checkboxes = forms.MultipleChoiceField(choices=THREE_CHOICES, widget=forms.CheckboxSelectMultiple)
```
And this is what it looks like when rendered with `{{ form|crispy }}` using the "bootstrap3" template pack:

![test form](https://cloud.githubusercontent.com/assets/8030396/4081286/39c86e9e-2ee6-11e4-8ec2-930d235e1031.png)

Notice that the checkboxes in the "Multiple checkboxes" portion of the form are too far to the left. The checkboxes actually get cut off if your browser is any narrower than 768 pixels.

**Comment by dbrgn** [comment]
There's also a possible CSS fix:

```css
form label.radio, form label.checkbox {
    font-weight: normal;
    padding-left: 20px;
}
```

**Comment by carltongibson** [comment]
Resolved in #358
